ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/Ds3ClientHelpersImpl.java

```diff
@@ -596,13 +593,22 @@ public ObjectStorageSpaceVerificationResult objectsFromBucketWillFitInDirectory(
      * Creates a folder in the specified bucket.
      */
     @Override
-    public PutFolderResponse createFolder(final String bucketName, final String folderName) throws IOException {
+    public PutObjectResponse createFolder(final String bucketName, final String folderName) throws IOException {
```

Comment:
I'm confused why we're returning the response.  As far as I'm aware, either the call worked, or it didn't.  If that's the case, return void, and let the exception bubble up to communicate the failure condition.